pom.xml

```diff
@@ -10,7 +10,7 @@
 	</parent>
 	<groupId>com.mykaarma</groupId>
 	<artifactId>urlshortener</artifactId>
-	<version>0.0.4</version>
+	<version>0.0.5</version>
```

Comment:
You also need to use this updated version in mkUrlShortener, otherwise your change won't be reflected in the service